core/src/main/java/org/dozer/BeanFactory.java

```diff
@@ -32,7 +32,8 @@
  *
  * <p>
  * <a
- * href="http://dozer.sourceforge.net/documentation/custombeanfactories.html">http://dozer.sourceforge.net/documentation/custombeanfactories.html</a>
+ * href="https://github.com/DozerMapper/dozer/blob/master/docs/asciidoc/documentation/custombeanfactories.adoc">
```

Comment:
Just for my better understanding - wthis will be updated to GitHub Pages links after #342 is implemented, right? If yes, maybe it makes sense to create a reminder issue to not forget about this.